predicators/approaches/vlm_planning_prompts/few_shot.txt

```diff
@@ -0,0 +1,38 @@
+You are highly skilled in robotic task planning, breaking down intricate and long-term tasks into distinct primitive actions.
+Consider the following skills a robotic agent can perform. Note that each of these skills takes the form of a `ParameterizedOption` and may have both discrete arguments (indicated by the `types` field, referring to objects of particular types),
+as well as continuous arguments (indicated by `params_space` field, which is formatted as `Box([<param1_lower_bound>, <param2_lower_bound>, ...], [<param1_upper_bound>, <param2_upper_bound>, ...], (<number_of_params>,), <datatype_of_all_params>)`).
+
+{options}
+
+
+Here are a number of example demonstrations of solving a range of goals from an expert demonstrator. For each of these examples, you are provided 
+with the goal the demonstrator was trying to achieve, the plan (i.e., sequence of skills with discrete and continuous arguments specified), as well
```

Comment:
In what format are you providing "the goal the demonstrator was trying to achieve"? 

If we want to stay as close to the original paper as possible, which does things like "Stack all the blocks", "Put all the blocks in the bottom corner" -- then we won't want to give ground atoms, but instead a description of the task.

Reply:
i think it's fine to just not bother here; it's quite annoying to change this and feels not worth it imo (we do have a notion of `EnvironmentTask`, which has a `GoalDescription` that's usually in natural language, but it's really annoying to bring this down into the approach